design-library/src/components/BccInfo/BccInfo.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import type { Component } from "vue";
+type Props = {
+  title: string;
+  infoIcon?: Component;
+  infoText?: string;
```

Comment:
Is the `info` prefix necessary here? Since it's just a prop on the component, keeping it the same as other components would have my preference, so just `icon` and `text`. Or maybe `text` can be a default slot as well.